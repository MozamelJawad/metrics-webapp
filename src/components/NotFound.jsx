import React from 'react';
import { Link } from 'react-router-dom';

function NotFound() {
  return (
    <div className="notfound">
      <h3>Page Not Found</h3>
      <p className="goTo"><Link to="/">Got to Home page</Link></p>
    </div>
  );
}

export default NotFound;
