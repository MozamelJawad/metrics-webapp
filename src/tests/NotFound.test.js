import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import NotFound from '../components/NotFound';

describe('Not Found Page: ', () => {
  it('Should render not found page correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <NotFound />
      </BrowserRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
