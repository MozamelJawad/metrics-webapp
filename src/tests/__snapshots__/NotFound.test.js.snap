// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page:  Should render not found page correctly 1`] = `
<div>
  <div
    style="color: red; text-align: center; margin-top: 1rem;"
  >
    <h3>
      Page Not Found
    </h3>
    <p
      style="margin-top: 1rem; color: blue;"
    >
      <a
        href="/"
      >
        Got to Home page
      </a>
    </p>
  </div>
</div>
`;
